data/data-pipeline/data_pipeline/etl/sources/census/etl.py

```diff
@@ -153,22 +156,22 @@ def _load_into_state_csvs(self, fips_code: str) -> None:
             None
         """
         ## write to individual state csv
-        geoid10_list = self.CBG_PER_STATE[fips_code]
+        tractd10_list = self.TRACT_PER_STATE[fips_code]
```

Comment:
Tiny nit: should this be `tractid10_list` instead of `tractd10_list`?

Reply:
Absolutely should be!